.github/workflows/ci.yml

```diff
@@ -81,10 +81,9 @@ jobs:
     - name: Install dev Dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install --upgrade setuptools
-        # We don't actually want to install briefcase; we just
-        # want the dev extras so we have a known version of tox.
-        python -m pip install $(ls dist/briefcase-*.whl)[dev]
+        python -m pip install --upgrade setuptools build wheel
```

Comment:
I'm not 100% clear on why `build` and `wheel` need to be on this list. Is this purely defensive, or is there a specific need?

Reply:
This script depends on `build` and `wheel`.

Reply:
/me facepalms. Of course - missed the imports at the top of the script.